### DISCUSSION TITLE
Custom SMTP 'from' other than own email

### DISCUSSION THREAD

**Discussion by hrbengtsen** [supabase/supabase#2135] [Questions] [answered]
Hey,

First time using custom SMTP, is there any way to change the from name to something else than my own email?

Also it would be nice for the SMTP password in the dashboard to be hidden.

**Comment by inian** [comment, marked as answer]
Hi @hrbengtsen, we have just implemented this. Follow this PR for updates - https://github.com/supabase/gotrue/pull/126

Good feedback on the hiding the SMTP passowrd, I have created an issue internally for it
